helm_deploy/whereabouts-api/values.yaml

```diff
@@ -1,18 +1,54 @@
 ---
 # Values here are the same across all environments
+generic-service:
```

Comment:
Hmmm lots of other projects seem to set `nameOverride` here - but it [defaults](https://github.com/ministryofjustice/hmpps-helm-charts/blob/654ca0e77a74eb847c0cd38dbaa6eb7492daa4e1/charts/generic-service/templates/_helpers.tpl#L6) to `.Chart.Name` so I think you're right to leave the default. We'll find out on deployment!